frontend/src/bundles/profile/pages/public-profile-page/public-profile-page.tsx

```diff
@@ -0,0 +1,185 @@
+import { actions as achievementsActions } from '~/bundles/achievements/store/achievements.js';
+import {
+    AchievementCard,
+    ActivityWidget,
+    ActivityWidgetColor,
+    Icon,
+    Loader,
+} from '~/bundles/common/components/components.js';
+import { IconColor } from '~/bundles/common/components/icon/enums/icon-colors.enum.js';
+import { IconName } from '~/bundles/common/components/icon/enums/icon-name.enum.js';
```

Comment:
Can we combine it?
 `import { IconColor, IconName  } from '~/bundles/common/components/icon/enums/enums.js';`